src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -293,21 +291,14 @@ protected function sortServiceIds(array $serviceIds)
     public static function getClassDescription(string $class, string &$resolvedClass = null): string
     {
         $resolvedClass = $class;
-
-        if (!interface_exists(DocBlockFactoryInterface::class)) {
-            return '';
-        }
-
         try {
             $r = new \ReflectionClass($class);
-            $resolvedClass = $r->name;
-
             if ($docComment = $r->getDocComment()) {
-                return DocBlockFactory::createInstance()
-                    ->create($docComment)
-                    ->getSummary();
+                preg_match_all('^\s*/\*\*\s+\*\s*(?<!@)(\w.+)^', $docComment, $matches);
```

Comment:
no need to match all, `if ($docComment && preg_match(...)) {` is fine.

Reply:
using ^ for a delimiter looks odd, i suggest `~regex~` or `{regex}`

Reply:
Changed to ~regex~ form. Does it look better now? I have tested it with local symfony 4.2.2 version and it works.